cmd/habitat/data_proxy/data_proxy.go

```diff
@@ -168,7 +232,19 @@ func (s *DataProxy) WriteHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func (s *DataProxy) AddDataNode(communityID string, dataNode *DataServerNode) error {
+func (s *DataProxy) AddSchemaHandler(w http.ResponseWriter, r *http.Request) {
+	return
```

Comment:
Is there a ticket for this? and maybe return 501 unimplemented for now?

Reply:
yeah i'm working on this now. i'll make a ticket if there's not one